### ISSUE TITLE
Why use the GSM8K test set questions as the few-shot input?

### ISSUE THREAD

**Issue by qtli** [wenhuchen/Program-of-Thoughts#8]
Thanks for your amazing work! 

I discovered that the few-shot examples utilized in [run_gsm8k.py](https://github.com/wenhuchen/Program-of-Thoughts/blob/main/run_gsm8k.py) are actually taken from the GSM8K test set. I'm curious why the questions from the training set weren't used as the few-shot examples instead.

Thanks for your help!

**Comment by wenhuchen** [comment]
I thought I removed it in the later version. Maybe I didn't update it. Sorry about that. But if you switch to some other training examples, the performance would be the same.

**Comment by qtli** [comment]
Thank you for your quick response!
